integration_tests/models/sql/test_get_column_values_relation_does_not_exist.sql

```diff
@@ -0,0 +1,29 @@
+{% set columns = dbt_utils.get_column_values(ref('test_model'), 'example_column', default='') %}
```

Comment:
```suggestion
{% set columns = dbt_utils.get_column_values(ref('test_model'), 'example_column', default=[]) %}
```

Not super important, but the default value should probably be a list. That lets you remove the `{% set columns = columns if columns is iterable else [] %}` line, which sort of masks the behavior that this test is trying to exercise!